### DISCUSSION TITLE
Extended Abstract

### DISCUSSION THREAD

**Discussion by joaoteixeira96** [joaomlourenco/novathesis#217] [Q&A] [answered]
Estou com dificuldade em adicionar um novo abstract de modo que fique desta ordem:

Abstract(en)
Extended Abstract(en)
Resumo(pt)

No template.tex adicionei esta linha por de baixo das 2 que lá estavam com abstract(en,pt)
\abstractfile[en]{extended-abstract-en}

Continua apenas a mostra os 2 abstracts que tinha antes.
Conseguem-me ajudar pff?

- versão FCT UNL
- Version 2020/02/18 [4.3.0]
- TeX Live version 2019
- Compiler pdfLatex (overleaf)

**Comment by joaomlourenco** [comment]
Please use english in this forum.
The template handles only *one* abstract for each language.  
I have to think what is the best way to dal with your request.  Give me a few days.  If I don't get back to you on this, please poke me here again.

**Comment by joaomlourenco** [comment, marked as answer]
@joaoteixeira96, please note that the version 4.3.0 is no longer supported and you can only deliver thesis/dissertations with the old covers until October 30.  So I strongly advise you to update your document to the template version 6.5.x.

**IMPORTANT:** Please let me know if you would like a solution for your current version (v4.3.0) or for v6.5.x!

**Reply by joaomlourenco** [reply]
I've got a (hacking) solution for you that works on the most recent version of the template, v6.5.2.

— In file "6_files.tex" add
```latex
\ntaddfile{abstract}[en]{extended-abstract}  % Extended abstract in English
```
— And start the file "extended-abstract.tex" with
```latex
\chapter*{Extended Abstract}
\phantomsection\pdfbookmark[1]{Extended Abstract}{bmexabstracten}
```
and then below add your text for the extended abstract.
